pytest_pootle/fixtures/import_export_fixtures.py

```diff
@@ -31,40 +31,32 @@ def file_import_failure(request):
 
 
 @pytest.fixture
-def ts_directory(request, tmpdir):
-    """Sets up a tmp directory with test ts files."""
+def ts_directory(po_directory, request, tmpdir, settings):
```

Comment:
cool - we can remove i think i just added to be sure that the env was to true to what would happen - ie the po files would already be there when the ts projects were set up.